test/modules/formats.js

```diff
@@ -5,15 +5,26 @@
     // This module tests whether our various file formats can be opened.
     // TODO: Add more file formats (with corresponding test data).
 
+    var viewer = null;
+
     QUnit.module('Formats', {
         beforeEach: function () {
             var example = document.createElement("div");
             example.id = "example";
             document.getElementById("qunit-fixture").appendChild(example);
+        },
+        afterEach: function () {
+            if ( viewer && viewer.close ) {
+                viewer.close();
+            }
+            if (viewer && viewer.destroy){
```

Comment:
Shouldn't destroy also close? Also, in what situation would `viewer` not have `destroy`?

Reply:
> Shouldn't destroy also close?

I wasn't entirely sure if removing `close` would have an unintended consequences or not. However, I've removed the call to `close` for all the tests where `destroy` is used and it doesn't seem to break the tests, so these have all been cleaned up.

Reply:
> Also, in what situation would `viewer` not have `destroy`?

I was just following the existing pattern of `if(viewer && viewer.close)` - but yes, `destroy` should always exist (as should `close`)

Reply:
Cool... It's not a big deal, but nice to clean up :)